12 add capability information automatically on build from the standard app api (#13)
materials-marketplace/user-docs: docs/source/apps/mp-api.md

To declare which capabilities are supported by your application, a REST-API that complies with the MarketPlace specification must be set for your application.
The API should be expressed using the [OpenAPI specification](https://www.openapis.org/) standard and be sent to the MarketPlace platform upon [registration](registration.md).

You can browse the official specification with redoc <a href="api.html" target="_blank">here</a>.
The complete specification is hosted publicly on [GitHub](https://github.com/materials-marketplace/standard-app-api).

## OpenAPI specification

> In this section we will explain some of the most relevant parts of the specification.

Reviewer: I think it's more standard to use simple present tense: (I added another change on the way)
```suggestion
In this section we explain some of the non-trivial parts of the specification.
```
